centreon/src/Core/Security/Authentication/Domain/Provider/OpenIdProvider.php

```diff
@@ -1061,10 +1039,10 @@ private function logAuthenticationInfo(string $message, ?array $content = null):
      */
     private function logExceptionInLoginLogFile(string $message, Exception $exception): void
     {
-        $this->centreonLog->insertLog(
-            CentreonUserLog::TYPE_LOGIN,
+        CentreonLog::create()->error(
+            CentreonLog::TYPE_LOGIN,
             sprintf(
-                "[Openid] [Error] {$message}",
+                "[Openid] {$message}",
                 $exception::class,
                 $exception->getMessage()
             )
```

Comment:
you need to pass the exception on argument of the error method

Reply:
You can pass the exception one time using the last argument. You don't need to pass the exception in the extra ;)

Reply:
Seen together, I resolve ! The next time, you can pass the exception in the extra options ;)